common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/io/impl/AbstractGeoInstanceWriter.java

```diff
@@ -65,6 +67,16 @@ public String getCustomEPSGPrefix() {
 		return getParameter(PARAM_CRS_CODE_FORMAT).as(String.class);
 	}
 
+	@Override
+	public void setWindingOrder(EnumWindingOrderTypes windingOrderType) {
+		setParameter(PARAM_UNIFY_WINDING_ORDER, Value.of(windingOrderType));
+	}
+
+	@Override
+	public EnumWindingOrderTypes getWindingOrder() {
+		return getParameter(PARAM_UNIFY_WINDING_ORDER).as(EnumWindingOrderTypes.class);
```

Comment:
This would be the ideal place to provide a default value, in case the parameter is not set. This should be something that classes extending the writer are able to adapt.
The default should be not doing anything, but for GML writers it should be a unification to CCW.